libs/features/pages/home/src/lib/main/home/home-page.component.html

```diff
@@ -1,3 +1,4 @@
 <div class="home-page">
   Elewa Education
-</div>
\ No newline at end of file
+</div>
+<elewa-website-elewa-project-item-card></elewa-website-elewa-project-item-card>
```

Comment:
No need to submit this part. The component does not need to be imported here. Any component that needs this will export it.